cumulus_library/schema/counts.py

```diff
@@ -54,45 +63,177 @@ def get_where_clauses(
     def get_count_query(
         self, table_name: str, source_table: str, table_cols: list, **kwargs
     ) -> str:
-        """Wrapper method for generating a counts table from a template"""
+        """Generates a counts table using a template
+
+        :param table_name: The name of the table to create. Must start with study prefix
+        :param source_table: The table to create counts data from
+        :param table_cols: The columns from the source table to add to the count table
+        :keyword where_clauses: An array of where clauses to use for filtering the data
```

Comment:
oh nice I didn't know about `keyword` as a convention

Reply:
yeah i just learned about this myself while i was looking into potential autogeneration of sphinx docs.